### DISCUSSION TITLE
How do I train a Chinese TTS model?

### DISCUSSION THREAD

**Discussion by godspirit00** [NVIDIA-NeMo/NeMo#4473] [Q&A] [answered]
I'd like to train a Chinese TTS model using the  [databaker dataset](https://www.data-baker.com/en/#/data/index/source). How do I start?

Thanks!

**Comment by redoctopus** [comment, marked as answer]
Consider checking out this tutorial about modifying FastPitch for training on a non-English dataset: https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

As well as this FastPitch/Mixer-TTS tutorial for more general information: https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/FastPitch_MixerTTS_Training.ipynb

**Reply by godspirit00** [reply]
Thanks! I'll have a try.
